packages/utils/src/lib/reports/generate-md-report.ts

```diff
@@ -139,7 +135,21 @@ export function aboutSection(
 
 export function reportPluginMeta({ plugins }: Pick<Report, 'plugins'>) {
   return {
-    headings: pluginMetaTableHeaders,
+    headings: [
+      {
+        key: 'plugin',
+        align: 'l' as const,
```

Comment:
You wouldn't need the `as const` here if the function used the `Table` return type. An explicit function return type will also give you better autocomplete in the `return` statement.

Reply:
removed